app/src/reference_counted_future_impl.h

```diff
@@ -25,7 +25,6 @@
 #include "app/src/include/firebase/internal/common.h"
 #include "app/src/mutex.h"
 
-
```

Comment:
nit: remove, unless it's important that this PR touches this file

Reply:
Done.